index.html

```diff
@@ -187,12 +187,13 @@
                 partial interface InputEvent {
                     readonly attribute DOMString     inputType;
                     readonly attribute DataTransfer? dataTransfer;
-                    sequence<StaticRange> getTargetRanges();
+                    sequence&lt;StaticRange&gt; getTargetRanges();
```

Comment:
@choniong So this line is reverting the change @foolip proposed a few days ago. Do you two agree on this change?